package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "@types/lodash": "^4.14.161",
+    "@types/react-map-gl": "^5.2.9",
```

Comment:
I thought react-map-gl now exports typescript definitions straight without this. Were there some definitions missing?

Reply:
At least for me it gave errors so I put this in 🤷

Reply:
Okay. Well it should not do any harm anyways :)